composer.json

```
@@ -19,16 +19,16 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": ">=7.0",
```

If we can't support any other major versions, then lets drop their support too.
```suggestion
        "php": "^7.1",
```